subsys/bluetooth/controller/hci_internal.c

```diff
@@ -181,6 +182,8 @@ static void supported_commands(sdc_hci_ip_supported_commands_t *cmds)
 
 #if defined(CONFIG_BT_CTLR_LE_ENC) && defined(CONFIG_BT_CTLR_LE_PING)
 	cmds->hci_set_event_mask_page_2 = 1;
+	cmds->hci_read_authenticated_payload_timeout = 1;
```

Comment:
Add below hci_le_test_end and above hci_le_set_data_length?

Reply:
moved